scripts/startTriggerService.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+export _JAVA_OPTIONS="-Xms8m -Xmx128m"
+
+set -e
+
+TRIGGER_SERVICE_HOST="127.0.0.1"
+TRIGGER_SERVICE_PORT=8088
+
+cleanup() {
+    pids=$(jobs -p)
+    echo Killing $pids
+    [ -n "$pids" ] && kill $pids
+}
+
+waitForTriggerService() {
+    until curl ${TRIGGER_SERVICE_HOST}:${TRIGGER_SERVICE_PORT}/livez
+    do
+        sleep 0.1
+    done
+}
+
+startTrigger() {
+    curl -X POST -H "Content-Type: application/json" \
+        -d "{ \"triggerName\": \"${1}\", \"party\": \"${2}\", \"applicationId\": \"my-app-id\" }" \
+        ${TRIGGER_SERVICE_HOST}:${TRIGGER_SERVICE_PORT}/v1/triggers
+}
+
+trap "cleanup" INT QUIT TERM
+
+if [ $# -lt 2 ]; then
+    echo "${0} SANDBOX_HOST SANDBOX_PORT [DAR_FILE]"
+    exit 1
+fi
+
+SANDBOX_HOST="${1}"
+SANDBOX_PORT="${2}"
+DAR_FILE="${3:-/home/daml/know-your-customer.dar}"
+
+PACKAGE_ID=$(daml damlc inspect ${3} | grep package | cut -d" " -f2)
+
+# Market setup DAML script
```

Comment:
Either the following or drop the comment:
```suggestion
echo "Market setup DAML script"
```

This applies to the other similar comments.

Reply:
The comments around triggers I would not touch, for me they are like section delimiters.

Reply:
Extract them into functions then. Just not comments. :)